src/shogun/machine/BaggingMachine.h

```diff
@@ -110,10 +51,13 @@ namespace shogun
 			/** get out-of-bag error
 			 * CombinationRule is used for combining the predictions.
 			 *
-			 * @param eval Evaluation method to use for calculating the error
+			 * @param eval Evaluation method to use for calculating the error. It can be
+			 * null. In such a case, default evaluation criteria will be used:
+			 * - Binary/Multiclass labels: Binary/Multiclass Accuracy;
```

Comment:
I see! I am not a big fan of this. Obscures the code, what about new cases, etc....
I'd rather have a factory or convenience method that instantiates the evaluation with the choices you made here, but not have that in the class itself